src/elm/Action.elm

```diff
@@ -773,7 +769,8 @@ type alias ClaimedAction =
 encodeClaimAction : ClaimedAction -> Encode.Value
 encodeClaimAction c =
     Encode.object
-        [ ( "action_id", Encode.int c.actionId )
+        [ ( "community_id", Eos.encodeSymbol c.communityId )
+        , ( "action_id", Encode.int c.actionId )
         , ( "maker", Eos.encodeName c.maker )
         , ( "proof_photo", Encode.string c.proofPhoto )
         , ( "proof_code", Encode.string c.proofCode )
```

Comment:
This call will likely fail because there is a new param for actions: `image`. This will allow us to add an image to actions, further helping identifying. Its not something we need to aim for in this PR, unless its easy to do. Either way, the param must be present, even if it is empty

Reply:
Looking at the contract, I don't see that param anywhere... Is it already added? The only parameter on that action seems to be `community_id`: https://github.com/cambiatus/contracts/pull/33/files#diff-d13b4469ad5e7c554df36836b6b572c3dd35952f853619e1fe69a949e1444996R161-R167